src/Parser/DeathKnight/Shared/RuneBreakdown.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+
+import ResourceBreakdown from 'Parser/Core/Modules/ResourceTracker/ResourceBreakdown';
+
+//Need to extend this class to change how passive rune regeneration is displayed.
+//The only new lines are 44, 49-54 and 78.
+class RuneBreakdown extends ResourceBreakdown {
+  static propTypes = {
+    tracker: PropTypes.object.isRequired,
+    showSpenders: PropTypes.bool,
+  };
+  
+  render() {
+    const { tracker, showSpenders } = this.props;
+    const resourceName = tracker.resourceName;
+    const generated = this.prepareGenerated(tracker.buildersObj);
+    const spent = this.prepareSpent(tracker.spendersObj);
+
+    let totalGenerated = tracker.generated;
+    let totalWasted = tracker.wasted;
+    
+    let totalSpent = tracker.spent;
+    let totalCasts = tracker.spendersCasts;
+
+    // looks wrong but totals are only for the purpose of percentage, and if nothing was wasted, then 0/1 gives correct result 0% wasted, if it's not 0 it retains its original value
+    totalGenerated = (totalGenerated === 0) ? 1 : totalGenerated;
+    totalWasted = (totalWasted === 0) ? 1 : totalWasted;
+
+    totalSpent = (totalSpent === 0) ? 1 : totalSpent;
+    totalCasts = (totalCasts === 0) ? 1 : totalCasts;
+
+    return (
+      <div>
+        <table className="data-table">
+          <thead>
+            <tr>
+              <th>Ability</th>
+              <th colSpan="2"><dfn data-tip="Runes generated from passive regeneration and abilities that accelerate it are estimates.">{resourceName} generated</dfn></th>
+              <th colSpan="2"><dfn data-tip="This is the amount of resources that were generated while you were already at cap.">{resourceName} wasted</dfn></th>
+            </tr>
+          </thead>
+          <tbody>
+            {generated && generated
+              .filter(ability => ability.abilityId === SPELLS.RUNE_1.id)
```

Comment:
I might be misreading but wouldn't this just show the data for rune 1?

Reply:
I just use that as the ID because it needs an ID to filter by, it has no functional use other than that. I merge all the runes into one in runetracker